sr/comp/knockout_scheduler/static_scheduler.py

```diff
@@ -13,6 +14,20 @@
 from .base_scheduler import BaseKnockoutScheduler, UNKNOWABLE_TEAM
 
 StaticMatchTeamReference = NewType('StaticMatchTeamReference', str)
+StaticMatchTeamReference.__doc__ = r"""
```

Comment:
Explicit assignment needed as `NewType`'s docstring otherwise seems to be what we get.